dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -295,27 +295,26 @@ else
-                      .\"data\".\"service.account.secret\" = \"\" | 
-                      .\"data\".\"service.account.id\" = \"\" | 
-                      .\"data\".\"che.jdbc.username\" = \"$RH_CHE_JDBC_USERNAME\" | 
-                      .\"data\".\"che.jdbc.password\" = \"$RH_CHE_JDBC_PASSWORD\" | 
-                      .\"data\".\"che.jdbc.url\" = \"$RH_CHE_JDBC_URL\" | 
-                      .\"data\".\"logs-encoding\" = \"plaintext\" " ${RH_CHE_CONFIG})
+CHE_CONFIG_YAML=$(yq ".\"data\".\"CHE_KEYCLOAK_REALM\" = \"NULL\" |
+                      .\"data\".\"CHE_KEYCLOAK_AUTH__SERVER__URL\" = \"NULL\" |
+                      .\"data\".\"CHE_KEYCLOAK_USE__NONCE\" = \"false\" |
+                      .\"data\".\"CHE_KEYCLOAK_CLIENT__ID\" = \"740650a2-9c44-4db5-b067-a3d1b2cd2d01\" |
+                      .\"data\".\"CHE_KEYCLOAK_OIDC__PROVIDER\" = \"https://auth.prod-preview.openshift.io/api\" |
+                      .\"data\".\"CHE_KEYCLOAK_GITHUB_ENDPOINT\" = \"https://auth.prod-preview.openshift.io/api/token?for=https://github.com\" |
+                      .\"data\".\"service.account.secret\" = \"\" |
+                      .\"data\".\"service.account.id\" = \"\" |
+                      .\"data\".\"che.jdbc.username\" = \"$RH_CHE_JDBC_USERNAME\" |
+                      .\"data\".\"che.jdbc.password\" = \"$RH_CHE_JDBC_PASSWORD\" |
+                      .\"data\".\"che.jdbc.url\" = \"$RH_CHE_JDBC_URL\" |
+                      .\"data\".\"CHE_LOG_LEVEL\" = \"plaintext\" " ${RH_CHE_CONFIG})
 
 CHE_CONFIG_YAML=$(echo "$CHE_CONFIG_YAML" | \
-                  yq ".\"data\".\"che-host\" = \"rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net\" |
-                      .\"data\".\"infra-bootstrapper-binary-url\" = \"http$SECURE://rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net/agent-binaries/linux_amd64/bootstrapper/bootstrapper\" |
-                      .\"data\".\"che-api\" = \"http$SECURE://rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net/api\" |
-                      .\"data\".\"che-websocket-endpoint\" = \"ws$SECURE://rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net/api/websocket\" |
+                  yq ".\"data\".\"CHE_HOST\" = \"rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net\" |
+                      .\"data\".\"CHE_INFRA_KUBERNETES_BOOTSTRAPPER_BINARY__URL\" = \"http$SECURE://rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net/agent-binaries/linux_amd64/bootstrapper/bootstrapper\" |
+                      .\"data\".\"CHE_API\" = \"http$SECURE://rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net/api\" |
+                      .\"data\".\"CHE_WEBSOCKET_ENDPOINT\" = \"ws$SECURE://rhche-$RH_CHE_PROJECT_NAMESPACE.devtools-dev.ext.devshift.net/api/websocket\" |
                       .\"metadata\".\"name\" = \"rhche\" |
-                      .\"data\".\"che-openshift-secure-routes\" = \"$RH_CHE_USE_TLS\" |
```

Comment:
@garagatyi could you clarify why TLS options are not needed in the script ?

Reply:
It is replaced with CHE_INFRA_OPENSHIFT_TLS__ENABLED

Reply:
hmm.. I got a bit lost why we had two properties "che-openshift-secure-routes" & "che-secure-external-urls"

Reply:
funny, looks like `che-secure-external-urls` is not used at all currently in the rh-che template

Reply:
yes, it is unused